### DISCUSSION TITLE
The login link is not working.

### DISCUSSION THREAD

**Discussion by Lipiniak92** [LiveHelperChat/livehelperchat#1979] [Q&A]
Yesterday, when attempting to log in, a message appeared stating that maintenance work was in progress. However, today, my login link redirects to your main website.

**Comment by remdex** [comment]
And what address it is?

**Reply by Lipiniak92** [reply]
https://sensen.livehelperchat.com/site_admin/

**Reply by remdex** [reply]
Most likely your instance has expired and is not active anymore.

**Reply by Lipiniak92** [reply]
I have the free version, does it always end?

**Reply by remdex** [reply]
Yes it does, but you can host it yourself always and pay nothing :)

**Reply by Lipiniak92** [reply]
and how to do it?

**Reply by Lipiniak92** [reply]
I have the possibility to install the Live Helper Chat application on the server with my hosting provider, is that enough?

**Reply by remdex** [reply]
Yes, that's enough :)

**Reply by Lipiniak92** [reply]
![image](https://github.com/LiveHelperChat/livehelperchat/assets/139580581/c1ffdce3-6847-45c4-b548-eb33cd96ed9e)

If I have a prestashop store installed on the given URL, will it still work if I install the application with the path as in the picture?

**Reply by remdex** [reply]
it will...
